client/signup/steps/site-info-collection/index.jsx

```diff
@@ -20,41 +17,31 @@ function getTypeformId() {
 }
 const Container = styled.div`
 	@media ( max-width: 960px ) {
-		margin: 0 20px 80px 20px;
+		margin: 0 20px 80px;
 	}
 	@media ( max-wdith: 425px ) {
 		margin-bottom: 180px;
 	}
 `;
 
-const LoadingContainer = styled.div`
-	align-items: center;
-	display: flex;
-	flex-direction: column;
-	height: 500px;
-	justify-content: center;
-`;
-
 function SiteInformationCollection( {
 	additionalStepData,
 	stepName,
 	submitSignupStep,
 	goToNextStep,
 } ) {
-	const translate = useTranslate();
 	const dispatch = useDispatch();
 
-	const { selectedDIFMDesign, selectedVertical } = useSelector( getSignupDependencyStore );
-	const username = useSelector( getCurrentUserName );
-	const [ isFormSubmitted, setIsFormSubmitted ] = useState( false );
+	const { selectedDesign, username: signupUsername } = useSelector( getSignupDependencyStore );
+	const loggedInUsername = useSelector( getCurrentUserName );
+
+	const selectedVertical = selectedDesign?.categories[ 0 ]?.name;
```

Comment:
I think it will be useful to drop in a comment here explaining that "0" index. Along with an explanation of the strict 1:1 (Non overlapping) mapping of the DIFM themes -> Categories.
Maybe a unit test can be useful in the theme page but not essential.

Reply:
@aneeshd16 ICYMI!